src/main/java/org/apache/maven/plugins/help/DescribeMojo.java

```diff
@@ -126,6 +128,20 @@ public class DescribeMojo extends AbstractHelpMojo {
     @Component
```

Comment:
`@Component` annotation should be removed and field can be final

Reply:
done